demo/2025-11/scripts/init.sh

```diff
@@ -0,0 +1,55 @@
+set -exo pipefail
+
+echo "Initializing WORKING_DIR"
+if [ ! -d "$WORKING_DIR" ]; then
+  mkdir "$WORKING_DIR"
+  echo "Working directory created: $WORKING_DIR"
+fi
+
+touch "$WORKING_DIR/.env"
+VARS_TO_DEFAULT=(
+  CARDANO_NODE
+  IMMDB_SERVER
+  SECONDS_UNTIL_REF_SLOT
+  REF_SLOT
+  DATA_DIR
+  LEIOS_MANIFEST
+  PYTHON3
+  ANALYSE_PY
+  PORT_UPSTREAM_NODE
+  PORT_NODE0
+  PORT_DOWNSTREAM_NODE
+  IP_UPSTREAM_NODE
+  IP_NODE0
+  IP_DOWNSTREAM_NODE
+  RATE_UP_TO_N0
+  DELAY_UP_TO_N0
+  RATE_N0_TO_UP
+  DELAY_N0_TO_UP
+  RATE_N0_TO_DOWN
+  DELAY_N0_TO_DOWN
+  RATE_DOWN_TO_N0
+  DELAY_DOWN_TO_N0
+)
+
+for var in "${VARS_TO_DEFAULT[@]}"; do
+  def_var="DEF_$var"
+  val=${!var:-${!def_var}}
```

Comment:
This is a bit odd, why don't we have defaults in the various scripts and just do overrides via `.env`? or why are there even `DEF_XXX` for `XXX` and not just set the vars to defaults .. by default :sweat_smile: ?